lib/FieldType/DataTransformer/CheckboxValueTransformer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\Core\FieldType\Checkbox\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class CheckboxValueTransformer implements DataTransformerInterface
```

Comment:
@adamwojs yes separate commits are good, we can always squash on merge if we don't need the history in master itself.